lib/sitemap-item.d.ts

```diff
@@ -1,77 +1,115 @@
 import builder = require('xmlbuilder');
+
+export declare interface NewsItem {
+  publication: {
+    name: string,
+    language: string
+  },
+  genres: string,
+  publication_date: string,
+  title: string,
+  keywords: string,
+  stock_tickers: string
+}
+
+export declare interface SitemapImg {
+  url: string,
+  caption: string,
+  title: string,
+  geoLocation: string,
+  license: string
+}
+
+export declare type yesno = 'yes' | 'no'
+export declare type allowdeny = 'allow' | 'deny'
+export declare type ChangeFrequency = 'always'|'hourly'|'daily'|'weekly'|'monthly'|'yearly'|'never'
+export declare interface VideoItem {
+  thumbnail_loc: string;
+  title: string;
+  description: string;
+  content_loc?: string;
+  player_loc?: string;
+  'player_loc:autoplay'
```

Comment:
Isn't this a typo ? Or it should be removed. Or Did you mean like this:

```
'player_loc:autoplay'?: string;
OR
player_loc?: 'autoplay';
```

Looks like the first one.